File: src/wsTripXML/Code/OTAClasses/IssueMCORS.vb
Option Strict Off
Option Explicit On

Imports System.Xml.Serialization

Namespace wsTravelTalk.wmIssueMCOOut

    '<remarks/>
    <System.Xml.Serialization.XmlRootAttribute(IsNullable:=False)>
    Public Class TT_IssueMCORS
        '<remarks/>
        Public Success As wmIssueMCOModels.Success

        '<remarks/>
        <System.Xml.Serialization.XmlArrayItemAttribute("Warning", IsNullable:=False)>
        Public Warnings() As String

        '<remarks/>
        <System.Xml.Serialization.XmlArrayItemAttribute("Error", IsNullable:=False)>
        Public Errors() As String

        '<remarks/>
        Public ConversationID As String

        '<remarks/>
        Public MCOs() As wmIssueMCOModels.MCOMask

        '<remarks/>
        <System.Xml.Serialization.XmlAttributeAttribute(),
         System.ComponentModel.DefaultValueAttribute(wmIssueMCOModels.Target.Production)>
        Public Target As wmIssueMCOModels.Target = wmIssueMCOModels.Target.Production

        '<remarks/>
        <System.Xml.Serialization.XmlAttributeAttribute()>
        Public TransactionStatusCode As wmIssueMCOModels.TransactionStatusCode

        '<remarks/>
        <System.Xml.Serialization.XmlAttributeAttribute()>
        Public EchoToken As String

        '<remarks/>
        <System.Xml.Serialization.XmlAttributeAttribute()>
        Public Version As Double

        '<remarks/>
        <System.Xml.Serialization.XmlIgnoreAttribute()>
        Public VersionSpecified As Boolean

        '<remarks/>
        <System.Xml.Serialization.XmlAttributeAttribute()>
        Public PrimaryLangID As String

        '<remarks/>
        <System.Xml.Serialization.XmlAttributeAttribute()>
        Public AltLangID As String


    End Class




End Namespace

File: src/wsTripXML/Code/OTAClasses/IssueMCORS.cs

using System.Xml.Serialization;

namespace wsTripXML.wsTravelTalk.wmIssueMCOOut
{

    // <remarks/>
    [XmlRoot(IsNullable = false)]
    public class TT_IssueMCORS
    {
        // <remarks/>
        public wmIssueMCOModels.Success Success;

        // <remarks/>
        [XmlArrayItem("Warning", IsNullable = false)]
        public string[] Warnings;

        // <remarks/>
        [XmlArrayItem("Error", IsNullable = false)]
        public string[] Errors;

        // <remarks/>
        public string ConversationID;

        // <remarks/>
        public wmIssueMCOModels.MCOMask[] MCOs;

        // <remarks/>
        [XmlAttribute()]
        [System.ComponentModel.DefaultValue(wmIssueMCOModels.Target.Production)]
        public wmIssueMCOModels.Target Target = wmIssueMCOModels.Target.Production;

        // <remarks/>
        [XmlAttribute()]
        public wmIssueMCOModels.TransactionStatusCode TransactionStatusCode;

        // <remarks/>
        [XmlAttribute()]
        public string EchoToken;

        // <remarks/>
        [XmlAttribute()]
        public double Version;

        // <remarks/>
        [XmlIgnore()]
        public bool VersionSpecified;

        // <remarks/>
        [XmlAttribute()]
        public string PrimaryLangID;

        // <remarks/>
        [XmlAttribute()]
        public string AltLangID;


    }




}